server/src/main/java/org/opensearch/action/search/SearchRequest.java

```diff
@@ -698,6 +707,15 @@ public String pipeline() {
         return pipeline;
     }
 
+    public SearchRequest resourceLimitGroupId(String resourceLimitGroupId) {
```

Comment:
```suggestion
    public SearchRequest setResourceLimitGroupId(String resourceLimitGroupId) {
```

Reply:
I had kept this method aligned with other methods in the class.